apps/examples/wc-vue3/vite.config.ts

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'vite';
+import vue from '@vitejs/plugin-vue';
+import vueJsx from '@vitejs/plugin-vue-jsx';
```

Comment:
Is this package necessary/recommended?

Reply:
It's an optional, suggested by `create-vue` CLI.

Seems like it's not that useful in this scenario, I'll remove it.